pyproject.toml

```diff
@@ -53,8 +46,13 @@ line-length = 140
 
 [tool.ruff.lint.per-file-ignores]
 "__init__.py" = ["F401"]
+"tests/*" = ["S101", "S105", "S106"]
```

Comment:
As you don't have any tests, this line is not required

Reply:
I'd like to leave that as is and supply tests with later PRs